sharding-core/sharding-core-common/src/main/java/org/apache/shardingsphere/core/rule/ShardingRule.java

```diff
@@ -85,36 +85,29 @@ public ShardingRule(final ShardingRuleConfiguration shardingRuleConfig, final Co
     }
     
     private Collection<TableRule> createTableRules(final ShardingRuleConfiguration shardingRuleConfig) {
-        Collection<TableRuleConfiguration> tableRuleConfigurations = shardingRuleConfig.getTableRuleConfigs();
-        Collection<TableRule> result = new ArrayList<>(tableRuleConfigurations.size());
-        for (TableRuleConfiguration each : tableRuleConfigurations) {
-            result.add(new TableRule(each, shardingDataSourceNames, getDefaultGenerateKeyColumn(shardingRuleConfig)));
-        }
-        return result;
+        return shardingRuleConfig.getTableRuleConfigs().stream()
+        .map(each -> new TableRule(each, shardingDataSourceNames, getDefaultGenerateKeyColumn(shardingRuleConfig)))
```

Comment:
with more blank in the front of this line.

Reply:
Ok,i will fix it